File: Roteiro_7/Modelo_aluno/Python/Aluno.py
class Aluno:
    def __init__(self, nome, idade, nota):
        self.nome = nome
        self.idade = idade
        self.nota = nota

    def Estudar(self):
        return f"{self.nome}'comecçou a estudar."

    def FazerProva(self):
        return f"A sua nota foi: {self.nota}..."
    
    def Colar(self):
        return f"{self.nome} 'esta colando"

Aluno1 = Aluno("Guilherme", 21, 4)
Aluno2 = Aluno("Julia", 21, 2)
Aluno3 = Aluno("Davi", 21, 0)

print(Aluno1.Estudar())
print(Aluno2.FazerProva())
print(Aluno3.Colar())

File: Roteiro_7/Modelo_aluno/Java/Aluno.java
public class Aluno {
    //atributos
    String nome;
    int idade;
    int nota;

    public Aluno (String nome, int idade, int nota){
        this.nome = nome;
        this.idade = idade;
        this.nota = nota;
    }
    public String Estudar(){
        return nome + " está estudando para prova ";
    }
    public String fazerProva(){
        return nome + " está fazendo uma prova ";
    }
    public static void main (String[]args){
        Aluno Aluno1 = new Aluno("Guilherme", 21, 4);
        Aluno Aluno2 = new Aluno("Julia", 21, 2);

        //acessando os atributos
        System.out.println("Nome: " + Aluno1.nome + " , idade: " + Aluno1.idade + " anos " + " , A nota dele é: " + Aluno1.nota);
        System.out.println("Nome: " + Aluno2.nome + " , idade: " + Aluno2.idade + " anos " + " , A nota dela é: " + Aluno2.nota);

        System.out.println(Aluno1.Estudar());
        System.out.println(Aluno2.fazerProva());
    }
}
